meta-resin-common/recipes-core/initrdscripts/files/ro-rootfs

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
 
 bootparam_ro="true"
+
+ro_enabled() {
```

Comment:
There may be a slight issue with how the dash from this script's filename is interpreted. I recall from the boot logs that it complains about ro_enabled not being available or something like that. But yes, we need to know if this has been tested